### PR TITLE
[WIP] Training Test

### PR DESCRIPTION
**Pull request by hua-phuoc-mulodo**
[training-test] - Add user feature spec.

### CODE REVIEW

**Comment by hua-phuoc-mulodo** [comment]
I added post feature spec: index, create. I will add update and destroy soon.

File: spec/features/post_feature_spec.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+feature "#Post management" do
+  scenario "index" do
+    visit '/'
+    expect(find('ul[@class="menu"]')).to have_link('Post')
+    find('ul[@class="menu"]').click_link("Post")
+  end
+
+  given!(:post)    { Post.create!(title: 'Post title 1', content: 'Post content 1.', quantity: 1) }
```
Comment (tran-hai-mulodo) [inline spec/features/post_feature_spec.rb]: @hua-phuoc-mulodo 
Please remove unuse space for this line. Thanks.

Comment (hua-phuoc-mulodo) [reply spec/features/post_feature_spec.rb]: Ok thanks.

Comment (hua-phuoc-mulodo) [reply spec/features/post_feature_spec.rb]: I removed unuse space.

**Comment by hua-phuoc-mulodo** [comment]
I add post_request_spec: index, show.

**Comment by tran-hai-mulodo** [comment]
@hua-phuoc-mulodo 
It's OK. I merged. Thanks.

**Pull request merged**